React/Exercise/src/data/data.ts

```diff
@@ -38,17 +38,54 @@ interface Ref {
       ]
     },
     {
-      link: 'user_clicks',
-      title: 'Exercise 2: Capturing User Clicks',
-      description: 'This exercise gets you started with event handling',
-      ref: [
-        {
-          name: 'How to center a text inside a div', 
-          url:'https://www.w3schools.com/css/css_align.asp', 
-        },
-        
+      link: 'custom_component',
```

Comment:
take this data and define them where they are being used, no need to create a separate file